apex_checks.xml

```diff
@@ -152,7 +152,7 @@
 
     <module name="AnnotationLocation">
       <property name="allowSamelineMultipleAnnotations" value="false"/>
-      <property name="allowSamelineSingleParameterlessAnnotation" value="true"/>
+      <property name="allowSamelineSingleParameterlessAnnotation" value="false"/>
```

Comment:
No, by default it applies to CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF. So it does not apply to PARAMETER_DEF.  It does not affect number of existing violations either.